api/db/migrations/20210920153452_add-column-focusedout-to-answers.js

```diff
@@ -0,0 +1,14 @@
+const TABLE_NAME = 'answers';
+const COLUMN_NAME = 'focusedOut';
+
+exports.up = function(knex) {
+  return knex.schema.table(TABLE_NAME, (table) => {
+    table.boolean(COLUMN_NAME).defaultTo(false);
```

Comment:
```suggestion
    table.boolean(COLUMN_NAME).notNullable().defaultTo(false);
```

Passer la colonne en `NOT NULL` éviterait d'avoir 3 états (`true`/`false`/`NULL`) alors qu'il me semble qu'on n'en a pas besoin.

Reply:
Ok !